### PR TITLE
less chatty Testnet1 outputs

### LINKED ISSUE #281: Cleanup testnet log
Default should be to log to console at WARN. Also the "Starting validation pipeline..."message is noisy.

### PR DESCRIPTION
**Pull request by sesam**
Aimed at #281

- "Starting validation pipeline for " -> "Processing "
- stdout logging default = Warning
- ERR -> INFO for "Transaction rejected: Already in pool" -- only for netadapter transaction_received, which (hopefully!) only gets tx from peers, and lots of those will be duplicates, that's good and not an error.

### CODE REVIEW

File: grin/src/adapters.rs [hunk 1]
```diff
@@ -58,7 +58,11 @@ impl NetAdapter for NetToChainAdapter {
 		);
 
 		if let Err(e) = self.tx_pool.write().unwrap().add_to_memory_pool(source, tx) {
-			error!(LOGGER, "Transaction rejected: {:?}", e);
+			if e.kind() == PoolError::AlreadyInPool {
+				info!(LOGGER, "Transaction rejected: {:?}", e);
```
Comment (ignopeverell) [inline grin/src/adapters.rs]: I think a debug here would be just fine. Thanks!

File: chain/src/pipe.rs [hunk 2]
```diff
@@ -53,7 +53,7 @@ pub fn process_block(b: &Block, mut ctx: BlockContext) -> Result<Option<Tip>, Er
 
 	info!(
```
Comment (ignopeverell) [inline chain/src/pipe.rs]: These can be changed to debug as well. There's a log for when the block is added (or errors when it fails) which should be enough.

File: grin.toml [hunk 3]
```diff
@@ -63,7 +63,7 @@ port = 13414
 log_to_stdout = true
 
 # Log level for stdout: Critical, Error, Warning, Info, Debug, Trace
-stdout_log_level = "Debug"
+stdout_log_level = "Warning"
```
Comment (cryptagoras) [inline grin.toml]: I would argue that `Info` should be the default at this stage, but minor thing.

Comment (kargakis) [reply grin.toml]: Agreed about "Info"

**Pull request merged**